Pipelines/Templates/Publish.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+jobs:
+- deployment: Publish
+  condition: and(succeeded(), eq(variables['Build.SourceBranchName'], variables['SigningBranch']))
+  continueOnError: false
+  displayName: Publish Packages to Feeds
+  pool:
+    name: Package ES Lab E
+  environment: 'SpatialAudio-Unity Release Approval'
+  strategy:
+    runOnce:
+      deploy:
+        steps:
+        - script: echo Syncing Source

```

Comment:
Just curious -- do these echos add much value? I haven't seen the debug spew, but I feel like it would be obvious which step the jobs are on.

Reply:
True, I think this is redundant. I'll remove in a separate PR.